src/config.ts

```diff
@@ -97,8 +97,8 @@ export type DeploymentStackConfig = CommonConfig & {
   };
   denySettings: {
     mode: "denyDelete" | "denyWriteAndDelete" | "none";
-    excludedActions: string[];
-    excludedPrincipals: string[];
```

Comment:
This change was necessary because of an accidental behavioral change to Stacks which would reject an empty array. I've also submitted a PR to fix the Stacks behavior, but it'll take some time before it is deployed.